providers/aws/resources/aws_rds.go

```diff
@@ -141,6 +143,30 @@ func (a *mqlAwsRds) getDbInstances(conn *connection.AwsConnection) []*jobpool.Jo
 	return tasks
 }
 
+type mqlAwsRdsDbinstanceInternal struct {
+	cacheSubnets *rdstypes.DBSubnetGroup
+	region       string
+}
+
+func (a *mqlAwsRdsDbinstance) subnets() ([]interface{}, error) {
+	if a.cacheSubnets != nil {
+		res := []interface{}{}
+		conn := a.MqlRuntime.Connection.(*connection.AwsConnection)
+		for i := range a.cacheSubnets.Subnets {
+			subnet := a.cacheSubnets.Subnets[i]
+			sub, err := NewResource(a.MqlRuntime, "aws.vpc.subnet", map[string]*llx.RawData{"arn": llx.StringData(fmt.Sprintf(subnetArnPattern, a.region, conn.AccountId(), convert.ToString(subnet.SubnetIdentifier)))})
+			if err != nil {
+				a.Subnets.State = plugin.StateIsNull | plugin.StateIsSet
+				return nil, err
+			}
+			res = append(res, sub)
+		}
+		return res, nil
+	}
+	a.Subnets.State = plugin.StateIsNull | plugin.StateIsSet
+	return nil, nil
```

Comment:
I'm wondering whether this should return an error.
I don't know this AWS service from personal use, so I skimmed over the docs. I found this one:
> Your VPC must have a DB subnet group that you create.

Could it be possible we cannot access this data?

Reply:
oh, interesting, ill check 👍

Reply:
ya, it seems we should always have this info